spec/rspec/core/output_wrapper_spec.rb

```diff
@@ -9,6 +9,11 @@ module RSpec::Core
       expect(output.string).to eq("message\nanother message").and eq(wrapper.string)
     end
 
+    it 'proxies IO instance methods with __send__ to be compatiable with IOs that redefine send' do
+      expect(output).to receive(:__send__).with(:puts, 'message')
```

Comment:
Please accept my apologies for not making myself clear.
Would it be possible to add a real-world case here that would fail with `send` but would work with `__send__`? `TCPSocket` or others?

Reply:
Thanks for looking at my PR @pirj!

How about this test?